docs-chef-io/content/server/v14/config_rb_server_optional_settings.md

```diff
@@ -1289,6 +1289,15 @@ This configuration file has the following settings for `opscode-erchef`:
 
     **New in Chef Infra Server 14.1**
 
+`opscode_erchef["allow_email_update_only_from_manage"]`
+
+:   Allow preventing user's from changing their e-mail with new allow_email_update_only_from_manage config.
```

Comment:
When set to true will allow users to update their email only from the manage console. When set to false users can change email from manage console as well as knife. (or something to this effect)